src/common/docsEnvironmentController.ts

```diff
@@ -29,6 +29,9 @@ export class DocsEnvironmentController implements EnvironmentController, vscode.
                 this.reloadWindow();
             } else if (event.affectsConfiguration(`${EXTENSION_NAME}.${USER_TYPE}`)) {
                 this._userType = this.getUserType();
+                if (this._userType === UserType.PublicContributor) {

```

Comment:
we can always send this event not matter what the new user type is since there is already userType in the event